### PR TITLE
#6683 - add new 'internal' field to differentiate between 3rd party a…

### PR DESCRIPTION
**Pull request by jvasquez75**
adding "internal"

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## v3.2.0
```
Comment (gdbrown) [inline CHANGELOG.md]: use `issue #blah: blah` format

File: aspects/wb-sponsors.xml [hunk 2]
```diff
@@ -27,6 +27,11 @@
       <title>List Badge</title>
       <validation datatype="html" allowedtags="*" max="64K"/>
     </meta>
+    <meta id="#internal">
```
Comment (gdbrown) [inline aspects/wb-sponsors.xml]: use `is-internal` as we're trying to make bools/flags more obvious by using "is", "has", "can" prefixes where possible.

**Comment by jvasquez75** [comment]
Updated

File: aspects/wb-sponsors.xml (hunk 2 above)
Comment (jvasquez75) [reply aspects/wb-sponsors.xml]: done

File: CHANGELOG.md (hunk 1 above)
Comment (jvasquez75) [reply CHANGELOG.md]: done

**Pull request merged**